app/components/auk/button.hbs

```diff
@@ -7,9 +7,26 @@
   >
     <Auk::Loader />
   </button>
+{{else if @route}}
+  <LinkTo
+    type="button"
+    class="auk-button {{this.skin}} {{this.size}} {{this.layout}} {{this.disabled}} {{this.block}}"
+    disabled={{@disabled}}
+    @route={{@route}}
+    @models={{this.models}}
+    @preventDefault={{@preventDefault}}
+    @query={{@query}}
+    @tagName="button"
+    ...attributes
+  >
+    <Auk::Button::InnerLayout @layout={{@layout}} @icon={{@icon}} @size={{@size}}>
+      {{yield}}
+    </Auk::Button::InnerLayout>
+  </LinkTo>
 {{else if @href}}
   {{!-- TODO: document in writing what the difference is between a "Auk::Button" with @href and a "Auk::ButtonLink" with @href --}}
```

Comment:
In het kader van je 2e punt in de PR beschrijving, wordt het antwoord op deze vraag (of de bredere context: button vs link) moeilijker dan het al was denk ik. Kan je deze meteen meenemen?

Reply:
Ik denk dat de toevoeging van LinkTo in de button component hierbuiten valt.
De "button vs link" discussie is eerder rond het gebruik van een `Auk::Button` met `@skin="borderless"` en het gebruik van een `Auk::ButtonLink`. Ongeacht of er een href, route of actie achterzit.

Die 2 zien er nagenoeg hetzelfde uit. En dat maakt het soms moeilijk om te weten welke van de 2 het design juist bedoeld.
VB: Beide worden doorheen onze applicatie gebruikt om de "annuleren" knop te tonen in modals/panels

Reply:
Daarbij volg ik @ValenberghsSven met dat de verschillen tussen die twee componenten vooral design-matig zijn. Nu, dat ter zijde, is het inderdaad soms verwarrend in welke context elk type moet gebruikt worden. Met de komst van een designer (om het werk van Mono op te volgen) zou ik dat ook uitgeklaard willen zien.